cwf/gateway/configs/ctraced.yml

```diff
@@ -11,3 +11,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 log_level: INFO
+
+# Directory for temporary storage of packet captures
+trace_directory: /var/opt/magma/tmp/trace
+
+# Interface to capture on
+trace_interface: eth0
```

Comment:
Why do we capture it only on this interface ? Is this meant to by overridden by some sort of a configuration ?

Reply:
Changed to multiple